pkg/s3/client.go

```diff
@@ -36,6 +37,7 @@ type Config struct {
 type FSMeta struct {
 	BucketName    string `json:"Name"`
 	Prefix        string `json:"Prefix"`
+	UsePrefix     string `json:"UsePrefix"`
```

Comment:
Why not make this a `bool`? `FSMeta` is encoded as json, so we can use proper types instead of encoding a bool as a string ourselves.